fearless/Common/View/RewardSelectionView.swift

```diff
@@ -3,10 +3,9 @@ import SoraUI
 
 class RewardSelectionView: BackgroundedContentControl {
     private(set) var titleLabel: UILabel!
-    private(set) var subtitleLabel: UILabel!
     private(set) var earningsTitleLabel: UILabel!
     private(set) var earningsSubtitleLabel: UILabel!
-    private(set) var iconView: UIImageView!
+    private(set) var iconView: UIButton!
```

Comment:
Why do we need button? Touch event is working on the whole view

Reply:
Button provides states out of the box, so we don't need to change images manually and store state